src/utils/api.js

```diff
@@ -270,6 +270,11 @@ const upload = (zipPath, sourceZipPath, appDir) => {
 
   const fullZipPath = path.resolve(appDir, zipPath);
   const fullSourceZipPath = path.resolve(appDir, sourceZipPath);
+  const isMissingZip = !fs.existsSync(fullZipPath);
+
+  if (isMissingZip) {
+    throw new Error('Missing a built app. Trying running `zapier push`.');
```

Comment:
Thinking about it, this is a little bit of an odd error message. I'd probably change it to "Try running `zapier build`", since push will have automatically run build and if they're seeing this message, it's because they're manually calling `upload` before `build`.

Also there's a typo: `Trying running`

Reply:
Typos... 

My thought is if they're running `upload` without `build`, there's a disconnect with CLI commands. What if we at least say something like `You need to run zapier build first. Or you could run zapier push, which will build and upload in one command.`

Reply:
Love it!